docs/rules/padding-line-between-tags.md

```diff
@@ -0,0 +1,168 @@
+---
+pageClass: rule-details
+sidebarDepth: 0
+title: vue/padding-line-between-tags
+description: Require or disallow newlines between sibling tags in template
+---
+# vue/padding-line-between-tags
+
+> Require or disallow newlines between sibling tags in template
+
+- :exclamation: <badge text="This rule has not been released yet." vertical="middle" type="error"> ***This rule has not been released yet.*** </badge>
+- :wrench: The `--fix` option on the [command line](https://eslint.org/docs/user-guide/command-line-interface#fixing-problems) can automatically fix some of the problems reported by this rule.
+
+## :book: Rule Details
+
+This rule requires or disallows newlines between sibling HTML tags
+
+<eslint-code-block fix :rules="{'vue/padding-line-between-tags': ['error']}">
+
+```vue
+<!-- ✓ GOOD -->
+<template>
+  <div>
+    <div></div>
+
+    <div>
+    </div>
+
+    <div />
+  </div>
+</template>
+```
+
+```vue
+<!-- ✗ BAD -->
+<template>
+  <div>
+    <div></div>
+    <div>
+    </div>
+    <div />
+  </div>
+</template>
+```
+
+</eslint-code-block>
+
+## :wrench: Options
+
+```json
+{
+  "vue/padding-line-between-tags": ["error", [
+    { "blankLine": "always", "prev": "*", "next": "*" }
+  ]]
+}
+```
+
+This rule requires blank lines between each sibling HTML tag by default.
```

Comment:
The default is now to check nothing at all: https://deploy-preview-1966--eslint-plugin-vue.netlify.app/rules/padding-line-between-tags.html#rule-details

I think it should be changed to require a blank space between all tags like noted here. Please also add a test case for that.

Reply:
I'm setting the default here [https://github.com/vuejs/eslint-plugin-vue/pull/1966/files/e210bc9f71b9dc7266a3b081d49430e3b638e5e3#diff-b8a55875caa63e9a2b2ac62113a599199f76d4f4f54a8f8c554a4582e2d412b4R68](url)

Should it be done differently?

There is also the first invalid test which tests the default behaviour [https://github.com/vuejs/eslint-plugin-vue/pull/1966/files/e210bc9f71b9dc7266a3b081d49430e3b638e5e3#diff-011f1880886531880d77f18276fbc8cbda2dd6954a775f62b318d6445c0be50aR150](url)

Reply:
Ah, you're right. Just the example is wrong, since it needs to be split into two separate code blocks containing only one `<template>` each.